README.md

```diff
@@ -1,30 +1,19 @@
-Confluent's Python Client for Apache Kafka<sup>TM</sup>
-=======================================================
+Confluent's Kafka Python Client
+===============================
 
-**confluent-kafka-python** is Confluent's Python client for [Apache Kafka](http://kafka.apache.org/) and the
-[Confluent Platform](https://www.confluent.io/product/compare/).
+**confluent-kafka-python** provides high-level Producer, Consumer and AdminClient classes compatible with all
```

Comment:
"classes" does not really mean anything, they could be empty.
Use "clients", "implementations", or something similar.

Reply:
yeah. that was a last minute change as it flowed better. i'll change it back to having no qualifier.